languages/elixir/concepts/basics/links.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "url": "https://elixir-lang.org/getting-started/basic-types.html#basic-arithmetic",
+    "description": "operators"
+  },
+  {
+    "url": "https://elixirschool.com/en/lessons/basics/documentation/#inline-documentation",
+    "description": "inline-documentation"
+  }
+]
```

Comment:
@ErikSchierboom What was the logic behind link extraction from `after.md` to here? I would have assumed simply all links from `after.md` made it to `links.json`, but in this specific instance, only 2 out of 6 were copied.

Reply:
Hmmm, must be an issue with my link scraping code 🤔 Guess we'll have to manually add them.